shared/src/commonMain/kotlin/world/respect/domain/opds/model/OpdsFeedMetadata.kt

```diff
@@ -0,0 +1,27 @@
+package world.respect.domain.opds.model
+
+import kotlinx.serialization.Serializable
+
+/**
+ * Represents metadata for an OPDS catalog or collection.
+ * This includes details like title, number of items, items per page, etc.
+ *
+ * For reference, see the schema: https://github.com/readium/webpub-manifest/blob/master/schema/metadata.schema.json
+ */
+@Serializable
+data class OpdsFeedMetadata(
+    val title: String,                     // Required field (title of the catalog or collection)
```

Comment:
@Prashant-Android This is missing the @type property as required by https://drafts.opds.io/schema/feed-metadata.schema.json . You can use Kotlinx serialization annotations if the field name in Json is not a valid variable name in Kotlin.

Reply:
Also missing identifier.

Reply:
Also missing subtitle property